docs/server/requirements.md

```diff
@@ -1,10 +1,10 @@
 # Server Requirements
 
-qooxdoo server is a basic component that runs in many different contexts and
-environments, as it has very little dependencies to the underlying runtime. For
-use in command-line tools and programs you will need a corresponding JavaScript
-interpreter like Node.js or Mozilla Rhino. For use in HTML5 Web Workers you will
-need a browser that supports this technology.
+qooxdoo server is a basic component that runs in many different contexts
+and environments, as it has very little dependencies to the underlying
```

Comment:
s/very little/very few/

In English, if things are countable, you use "few", e.g., "There I saw a few clouds."
If they are not easily countable, you use "little", e.g., "There is very little water in that glass." or even "There is only a little sand in the box." (but if reworded to be countable... "There are only a few grains of sand in the box.")